sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BroadcastHashJoinExec.scala

```diff
@@ -186,8 +186,7 @@ case class BroadcastHashJoinExec(
    */
   private def getJoinCondition(
       ctx: CodegenContext,
-      input: Seq[ExprCode],
-      anti: Boolean = false): (String, String, Seq[ExprCode]) = {
```

Comment:
I like this change.

Reply:
so we never set it to true?

Reply:
We used to in https://github.com/apache/spark/pull/19324/files#diff-4455c05ddcdb096c36d9e0bd326dfe12L389, we don't anymore.

Reply:
nvm, I saw the refactor in `codegenAnti`, cool!